addon/templates/components/frost-modal-input.hbs

```diff
@@ -1,11 +1,35 @@
-{{#if titleComponent}}
-  {{component titleComponent}}
-{{else}}
-  <div class="input-header">
-    <div class="primary-title">{{title}}</div>
-    {{#if subtitle}}
-      <div class="sub-title">{{subtitle}}</div>
-    {{/if}}
-  </div>
-{{/if}}
-{{yield}}
+{{#ember-remodal
+  buttonClasses='frost-button'
+  forService=true
+  openButton=openButton
+  openButtonClasses=openButtonClasses
+  closeOnOutsideClick=false
+  closeOnEscape=false
+  disableNativeClose=true
+  name=modalName as |modal|}}
+  {{#if titleComponent}}
+    {{component titleComponent}}
+  {{else}}
+    <div class="input-header">
+      <div class="primary-title">{{title}}</div>
+      {{#if subtitle}}
+        <div class="sub-title">{{subtitle}}</div>
+      {{/if}}
+    </div>
+  {{/if}}
+  {{yield}}
+  {{#yield-slot 'form'}}
+    {{#perfect-scroll
```

Comment:
tried to add this here but now the modal isnt rendering. Technically, height is 0. Form the console, Header and actions are in the source but the form component that is supposed to be yielded here isnt rendered :(